hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/audit/impl/ActiveAuditManagerS3A.java

```diff
@@ -82,10 +81,11 @@
  * will deactivate the wrapped span and then
  * switch the active span to the unbounded span.
  *
- * The inner class {@link AWSAuditEventCallbacks} is returned
- * as a request handler in {@link #createRequestHandlers()};
- * this forwards all requests to the outer {@code ActiveAuditManagerS3A},
- * which then locates the active span and forwards the request.
+ * This class also implements {@link ExecutionInterceptor} and
+ * returns itself in {@link #createExecutionInterceptors()};
+ * once registered with the S3 client, the implemented methods
+ * will be called on lifetime events for S3 requests,
+ * which then locate the active span and forward the request.
```

Comment:
I was a little confused by how we describe 'lifetime events'.

Maybe we do something like:

```suggestion
 * This class also implements {@link ExecutionInterceptor} and
 * returns itself in {@link #createExecutionInterceptors()};
 * once registered with the S3 client, the implemented methods
 * will be called during different parts of an SDK request lifecycle,
 * which then locate the active span and forward the request.
```